tests/test-vectors.js

```diff
@@ -765,8 +765,27 @@ var generalVectors = [
 	id: 37,
 	input: "<div style=\"color:))))) green\">parenthesis test 5</div>",
 	output: "<div>parenthesis test 5</div>"
+},
+{
+	id: 38,
+	input: "<img />",
```

Comment:
u need one more test case - `<img />` and `<img/>` are different according to the spec.